app/cdap/services/ThemeHelper.ts

```diff
@@ -435,6 +435,7 @@ function parse1Point0Spec(themeJSON: IOnePoint0SpecJSON): IThemeObj {
       showReloadSystemArtifacts: true,
       showSqlPipeline: true,
       showCDC: false,
+      onPremTetheredInstance: false,
```

Comment:
We would want to show the "Resources" tab on HDF in both tethered and non-tethered mode. 
Should this be onPremInstance instead of onPremTetheredInstance?

Reply:
This maps to the field `on-prem-tethered-instance` in the theme file: https://github.com/cdapio/cdap-ui/blob/develop/app/cdap/services/ThemeHelper.ts#L549 I'd like to keep those 2 names in sync, and I don't think we can change the name in the theme file because that would affect instances that are already deployed.